app/controllers/quiz_controller.rb

```diff
@@ -2,15 +2,16 @@ class QuizController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def index
-    puts "{ original_url => #{request.original_url}, ip => #{request.ip}"
+    puts "{ original_url => #{request.original_url}, ip => #{request.remote_ip}"
   end
 
   def task
     st=Time.now
     answer=nil
     case level = params["level"].to_i
     when 1
-      answer = $data_1[Unicode::downcase(params["question"].del_punc)]
+      answer = $data_1[params["question"].del_punc]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.